### PR TITLE
wasm

### PR DESCRIPTION
**Pull request by iunanua**
### Checklist

- [ ] The CHANGELOG.md has been updated
- [ ] Unit tests have been updated and pass
- [ ] If known, an appropriate milestone has been selected

### CODE REVIEW

File: benchmark/babel-rewrite.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const babelParser = require('@babel/parser')
```
Comment (CarlesDD) [inline benchmark/babel-rewrite.js:1]: I'm not sure if we must add the Licensing header in benchmark files.

File: main.js [hunk 2]
```diff
@@ -3,16 +3,20 @@
  * This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
  **/
 'use strict'
-
 const { getPrepareStackTrace } = require('./js/stack-trace/')
 const { cacheRewrittenSourceMap } = require('./js/source-map')
 
+const nativeRewriter = process.env.NATIVE_REWRITER === 'true'
```
[the remark this answers is not in the document]
Comment (iunanua) [reply main.js]: It is not needed but allows us to load the previous native Rewriter for testing purposes for example. I renamed it as you suggested but I can remove it if bother us...

File: main.js [hunk 3]
```diff
@@ -32,7 +36,12 @@ class CacheRewriter {
 
 function getRewriter () {
   try {
-    NativeRewriter = require('node-gyp-build')(__dirname).Rewriter
+    const iastRewriter = nativeRewriter ? require('node-gyp-build')(__dirname) : require('./wasm/native_iast_rewriter')
```
[the remark this answers is not in the document]
Comment (iunanua) [reply main.js]: Renamed

File: benchmark/babel-rewrite.js (hunk 1 above)
Comment (iunanua) [reply benchmark/babel-rewrite.js:6]: Added

File: .github/workflows/build-wasm.yml [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+  test-native-js:
+    needs: ['wasm-pack']
+    strategy:
+      matrix:
+        target:
+          - name: linux 64 bits
+            os: ubuntu-latest
+            artifact: linux-x64-gnu
+          - name: windows 64 bits
```
Comment (uurien) [inline .github/workflows/build-wasm.yml:81]: Can we try windows 32 bits?

File: .github/workflows/build-wasm.yml [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - run: npm ci --ignore-scripts
+      - uses: codex-team/action-nodejs-package-info@v1
+        id: package
+      - uses: actions/download-artifact@v3
+        with:
+          name: datadog-native-iast-rewriter-${{ steps.package.outputs.version }}
+      - run: npm i --verbose datadog-native-iast-rewriter-${{ steps.package.outputs.version }}.tgz
```
Comment (uurien) [inline .github/workflows/build-wasm.yml:102]: why `--verbose`?

Comment (iunanua) [reply .github/workflows/build-wasm.yml:102]: Possibly for debugging reasons...

File: package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@datadog/native-iast-rewriter",
   "homepage": "https://github.com/DataDog/dd-native-iast-rewriter-js/blob/main/README.md",
-  "version": "0.1.0-beta.3",
+  "version": "1.0.0",
```
Comment (uurien) [inline package.json:4]: `pre` to merge in master, and we will change it in `v1.x` branch.
```suggestion
  "version": "1.0.0-pre",
```

**Review by uurien — APPROVED** [review]
Change the version in package and 👍

File: .github/workflows/build-wasm.yml (hunk 4 above)
Comment (iunanua) [reply .github/workflows/build-wasm.yml:81]: done

**Pull request merged**